anesthetic/samples.py

```diff
@@ -58,6 +58,12 @@ class MCMCSamples(WeightedDataFrame):
         Anything equal or below this value is set to `-np.inf`.
         default: -1e30
 
+    discard_burnin: int or float
```

Comment:
Any strong feelings about the naming of the variable? 
Happy to change if something else would be more appropriate.

Reply:
Same goes more generally for the whole docstring.

Reply:
changed to `burn_in`